docs/user-guide/metrics.md

```diff
@@ -21,6 +21,15 @@ For example `http://<styx-host>/admin/metrics/requests.response` would return
     requests.response.status.2xx
     ...
     
+## Searching Metrics for string
+
+To see all metrics containing a given string, a query parameter can be provided.
+This works with both specific metric roots and the set of all metrics:
+
+`http://<styx-host>/admin/metrics?search=<term>`
+
+`http://<styx-host>/admin/metrics/<metric-name>?search=<term>`
+
```

Comment:
My immediate question is "what is `<term>`"? Please elaborate this. 

Also, I would use more straightforward language:

> Use a `search` query parameter to filter for metrics names matching a given string. For example `search=count` only shows metrics whose name contains `count`. The filtering is applied to the results of the metrics query.

Reply:
Also I am being very pedantic now but `filter` would be more accurate name for the query parameter, instead of `search`. WDYT?